templates/app-form-details.html

```diff
@@ -0,0 +1,229 @@
+    <h2 class="large-white-text">Application Details</h2>
+  </div>
+  <div class="row row__center">
+    <div class="column">
+      <h3>The Program</h3>
+      <p class="justify">
+        Techtonica’s full-time, virtual software engineering training program
+        provides the opportunity seeking economic empowerment to learn technical
+        skills in a safe, inclusive space and start their first jobs in tech. We
+        select participants who are independent but good at collaborating,
+        motivated to succeed, committed to learning to code, flexible, curious,
+        open-minded, and excited for long-term careers in the field, have a good
+        attitude and a growth mindset, can calmly manage extended periods of
+        frustration (troubleshooting is something you'll face forever in tech!),
+        and want to pay it forward.
+        <br />
+        We aim to empower those most underrepresented in tech and to help tech
+        become more diverse and inclusive. To that end, we have been very
+        intentional about creating a safe community, building a network, and
+        providing personalized support, which means open communication is
+        absolutely necessary. If a special situation comes up, we need you to
+        let us know so we can sympathize and brainstorm solutions with you.
+        We've had participants lose housing, have their laptop stolen, become
+        pregnant, face abuse, and more, and if we don't know about it, we can't
+        help.
+        <br />
+        88% of our graduates so far are BIPOC, 21% have a disability, 21% are
+        parents, 10% identify outside the gender binary, and 1% are veterans.
+        Our cohorts have had between 8 and 18 participants so far, and income
+        increase is usually 3-5x. The
+        participant-to-technical-program-staff-member ratio has historically
+        been between 5:1 and 9:1. We appreciate an average of 98 hours of
+        volunteer support per participant.
+      </p>
+      <h3>Pre-Req and Form Deadline</h3>
+      <p>
+        Lorem ipsum odor amet, consectetuer adipiscing elit. Et per commodo
+        litora duis neque congue. Convallis quisque suspendisse et est nullam
+        ligula magna habitant conubia. Ad torquent tortor etiam donec; fermentum
+        nascetur malesuada maximus. Amet enim magnis porta nec, donec ac aliquam
+        molestie mollis. Volutpat neque magna malesuada malesuada dolor
+        tristique. Accumsan feugiat varius tortor convallis aenean ridiculus.
+        Dui penatibus gravida sem ipsum hendrerit pulvinar tincidunt efficitur.
+      </p>
+      <h3>Eligibility</h3>
+      <p>
+        You are eligible to apply for Techtonica’s full-time program if you:
+      </p>
+      <ul>
+        <li>
+          are a woman (cis OR trans) or non-binary adult. We especially
+          encourage Latinx, Black, and non-binary folks to apply.
+        </li>
```

Comment:
This is an excellent opportunity to implement some conditional logic and cut down on the walls of eligibility text and form fields. A gender identity question can be used to quickly disqualify anyone whose gender identity does not line up with the eligibility requirements, saving them an immense amount of time on a form they do not need to fill out. If they answer with an eligible value, move them onto the rest of the form. If they answer with a non-eligible value, show them copy similar to this and send them off on their way.